pkg/objstore/oss/oss.go

```diff
@@ -0,0 +1,349 @@
+package oss
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math"
+	"math/rand"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	alioss "github.com/aliyun/aliyun-oss-go-sdk/oss"
+
```

Comment:
Please kill this newline and run `make format` (Current tooling limitation)

Reply:
OK